src/Pact/Types/Command.hs

```diff
@@ -95,7 +102,7 @@ data Command a = Command
   , _cmdSigs :: ![UserSig]
   , _cmdHash :: !PactHash
   } deriving (Eq,Show,Ord,Generic,Functor,Foldable,Traversable)
-instance (Serialize a) => Serialize (Command a)
+-- instance (Serialize a) => Serialize (Command a)
```

Comment:
This instance seemed unused, and it's very risky to provide this instance, but I don't want to break downstream if I don't have to.